file-formats/chkv/examples/z_aff_example_chkv.chkv.json

```diff
@@ -52,7 +50,7 @@
               "sign": "exclude",
               "option": "equals",
               "low": "0002470721",
-              "high": "0000000000"
+              "high": "0000000001"
```

Comment:
Does it make sense to have a Lowe `high` value than `low`

Reply:
We left the "high" field empty